examples/contract.rs

```diff
@@ -12,7 +12,10 @@ async fn main() -> web3::contract::Result<()> {
 
     let my_account = hex!("d028d24f16a8893bd078259d413372ac01580769").into();
     // Get the contract bytecode for instance from Solidity compiler
-    let bytecode = include_str!("./res/contract_token.code");
+    let bytecode = include_str!("./res/contract_token.code")
+        .to_string()
+        .trim_end()
+        .to_owned();
```

Comment:
```suggestion
    let bytecode = include_str!("./res/contract_token.code").trim_end();
```
Is it really necessary to convert to an owned `String`? I think `&str` should work just fine.